packages/grid/x-grid-modules/src/hooks/features/useComponents.tsx

```diff
@@ -10,6 +10,7 @@ import {
   Rows,
 } from '../../models';
 import { LoadingOverlay, NoRowMessage } from '../../components';
+import { ErrorMessage } from '../../components/error-message';
```

Comment:
Regarding the convention for components, we use the name of the main export.

We might not want to migrate the other files yet, but at least having the new ones following the convention will avoid extra work in the future :)

```suggestion
import { ErrorMessage } from '../../components/ErrorMessage';
```

Reply:
I will do this one in another PR with all files.